godot_project/editor/controls/dockers/workspace_docker/simulations_docker/simulations_docker.tscn

```diff
@@ -1,18 +1,8 @@
-[gd_scene load_steps=2 format=3 uid="uid://bystdhbca4tt8"]
+[gd_scene load_steps=3 format=3 uid="uid://cyubgn1obyavu"]
 
 [ext_resource type="Script" uid="uid://dpbmxpgwgmg6c" path="res://editor/controls/dockers/workspace_docker/simulations_docker/simulations_docker.gd" id="1_2u767"]
+[ext_resource type="PackedScene" uid="uid://bot7rbc830g06" path="res://editor/controls/dockers/workspace_docker.tscn" id="1_6f3if"]
```

Comment:
This makes the simulation docker inherit from the base docker scene instead of recreating its own scene tree.